arch/arm64/configs/bcmrpi3_defconfig

```diff
@@ -1272,6 +1272,7 @@ CONFIG_CRYPTO_TGR192=m
 CONFIG_CRYPTO_WP512=m
 CONFIG_CRYPTO_CAST5=m
 CONFIG_CRYPTO_DES=y
++CONFIG_CRYPTO_LZ4=m
```

Comment:
This doesn't look right to me - have you included a `+` at the start of your added line?

Reply:
Yes. Sorry. It wasn't seeming to let me submit without it :/